src/images/InputErrorValidationImage.jsx

```diff
@@ -0,0 +1,60 @@
+import * as React from "react"
+import {ErrorValidation} from "../components/Errors/ErrorValidation/ErrorValidation";
+
+class SvgComponent extends React.Component {
+    constructor(props) {
+        super(props);
+
+        this.state = {
+            flagMouse: false
+        }
+    }
+
+    onChangeMouseFlag = (flag) => {
+        this.setState({
+            flagMouse: flag
+        });
+    };
+
+    render() {
+        return (
+            <div style={{position: 'absolute',  right: '16%'}}>
```

Comment:
I suggest putting the SVG image into a separate `.svg` file in static folder

Reply:
it will be problematic to do this.
 I don’t remember where I found it before generating it in react